govcd/common_test.go

```diff
@@ -672,7 +672,7 @@ func deleteVapp(vcd *TestVCD, name string) error {
 // makeEmptyVapp creates a given vApp without any VM
 func makeEmptyVapp(vdc *Vdc, name string) (*VApp, error) {
 
-	err := vdc.ComposeRawVApp(name)
+	err := vdc.ComposeRawVApp(name, name)
```

Comment:
Should the second parameter hold desription?

Reply:
This is a testing function that is not using the description. Since I changed the signature of `ComposeRawVapp`, I added the vApp name as description.

Reply:
Nit, but you could use something like `"Description of " + name` - this way in the unlikely case a person would look into this populated field, it would be clear that it works and is not a bug.

Reply:
Changed.
However, the life span of a vApp created by this function is usually measured in minutes. The likelihood of someone looking into it is negligible.

Reply:
That's why it's a nit :) BTW, I think gofmt is needed there.